scripts/install.sh

```
@@ -60,71 +105,211 @@ case "$version" in
 
 tmpdir="$(mktemp -d)"
 trap 'rm -rf "$tmpdir"' EXIT
-dmg_path="${tmpdir}/con.dmg"
+
+if [ "$os" = "macos" ]; then
+  archive_path="${tmpdir}/con.dmg"
+else
+  archive_path="${tmpdir}/con.tar.gz"
+fi
 
 printf "   ${DIM}·${R}  downloading"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)